src/satosa/exception.py

```diff
@@ -68,7 +68,9 @@ def message(self):
         """
         return self._message.format(error_id=self.error_id)
 
-
+class UnknownSystemEntity(SATOSAError):
```

Comment:
I can already tell that is going to be useful for making the saml bits optional, but in this changeset this exception is not used.

Reply:
OK